yams_py_tests/test_meta.py

```diff
@@ -0,0 +1,12 @@
+import re
+from pathlib import Path
+from importlib.metadata import version
+from yams_py import Patcher
+
+
+def test_correct_versions():
+    cs_version = Patcher.get_lib_version()
+    python_version = version("yams-py")
```

Comment:
isn't the name of the package `yams_py` according to setup.cfg?

Reply:
I think this one's supposed to be with a hyphen.
While 
```
print("yams_py: " + str(version("yams_py")))
print("yams-py: " + str(version("yams-py")))
```
prints the same for me (`0.0.1`), tests fail with `yams_py` as that one returns MAJOR.MINOR.BUILD.REVISION instead of MAJOR.MINOR.BUILD

Reply:
Nevermind, I'm dumb, had a local build still of yams that still returned the major.minor.build.revision which was why tests failed for me locally.
Will change to underscore for consistency